_includes/nav.html

```diff
@@ -58,4 +58,17 @@
       <a href="{{ link.url | absolute_url }}" class="navigation-list-link" style="font-size: 14px;"">{{ link.title }}</a>
     </li>
   {%- endfor -%}
+  <br>
+  <br>
+  &nbsp;&emsp;&emsp;<a class="github-button" href="https://github.com/konveyor/move2kube/subscription" data-icon="octicon-eye" data-show-count="true" aria-label="Watch konveyor/move2kube on GitHub">Watch</a>
+  <br>
+  &nbsp;&emsp;&emsp;<a class="github-button" href="https://github.com/konveyor/move2kube" data-icon="octicon-star" data-show-count="true" aria-label="Star konveyor/move2kube on GitHub">Star</a>
+  <br>
+  &nbsp;&emsp;&emsp;<a class="github-button" href="https://github.com/konveyor/move2kube/fork" data-icon="octicon-repo-forked" data-show-count="true" aria-label="Fork konveyor/move2kube on GitHub">Fork</a>
+  <br>
+  &nbsp;&emsp;&emsp;<a class="github-button" href="https://github.com/konveyor/move2kube/archive/master.zip" data-icon="octicon-download" aria-label="Download konveyor/move2kube on GitHub">Download</a>
+  <br>
+  &nbsp;&emsp;&emsp;<a class="github-button" href="https://github.com/konveyor/move2kube/issues" data-icon="octicon-issue-opened" aria-label="Issue konveyor/move2kube on GitHub">Issue</a>
+
+  <script async defer src="https://buttons.github.io/buttons.js"></script>
```

Comment:
Let's retain only the star and fork, and lets try to make it appear side by side. Lets remove others.

Reply:
Okay sure.